Automattic/jetpack
# VideoPress Block: Add UI for Customizing Player Seekbar Colors

Adds a new panel to the VideoPress block to set custom colors for the seekbar:

<img width="956" alt="Screen Shot 2020-12-18 at 2 33 54 PM" src="https://user-images.githubusercontent.com/789137/102663436-1a4d8d80-413e-11eb-8c5e-e2cf6661114d.png">

Props to @pgk for building the majority of this PR :) 

Note that the iframe will reload everytime you choose a color... this isn't ideal and we are looking into improving the oembed to work via JS instead of an iframe for scenarios like this.

Fixes #

#### Changes proposed in this Pull Request:
Adds a new panel to set the seek bar colors for main, loading, and played colors. Every time you change a color, the player iframe gets reloaded so that the new color changes will show.

#### Jetpack product discussion
pxWta-Tk-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Add or edit a video block, don't choose any custom seekbar colors yet. They should have the usual grey/black combo.
* Now choose some seekbar colors, click `Save colors`, and the player should refresh with the new colors.
* Publish the post and view it on the front-end, the custom colors should also show.

#### Proposed changelog entry for your changes:


* VideoPress Block: Added support for custom video player seekbar colors.
